plenum/server/node.py

```diff
@@ -2172,143 +2151,21 @@ def doStaticValidation(self, request: Request):
         if TXN_TYPE not in operation:
             raise InvalidClientRequest(identifier, req_id)
 
-        self.execute_hook(NodeHooks.PRE_STATIC_VALIDATION, request=request)
         if operation[TXN_TYPE] != GET_TXN:
             # GET_TXN is generic, needs no request handler
             txn_type = operation[TXN_TYPE]
             req_manager = self._get_manager_for_txn_type(txn_type)
             if req_manager is None:
-                # TODO: This code should probably be removed.
-                if self.opVerifiers:
-                    try:
-                        for v in self.opVerifiers:
-                            v.verify(operation)
-                    except Exception as ex:
-                        raise InvalidClientRequest(identifier, req_id) from ex
-                else:
-                    raise InvalidClientRequest(identifier, req_id, 'invalid {}: {}'.
-                                               format(TXN_TYPE, operation[TXN_TYPE]))
+                raise InvalidClientRequest(identifier, req_id, 'invalid {}: {}'.
+                                           format(TXN_TYPE, operation[TXN_TYPE]))
             else:
                 req_manager.static_validation(request)
 
-        self.execute_hook(NodeHooks.POST_STATIC_VALIDATION, request=request)
```

Comment:
Did we remove these hooks at all?

Reply:
Yes, for now the next hooks will be used:
in node:
* NodeHooks.PRE_SIG_VERIFICATION
in replica:
* ReplicaHooks.CREATE_PPR
* ReplicaHooks.APPLY_PPR
* ReplicaHooks.CREATE_PR
* ReplicaHooks.CREATE_ORD